### PR TITLE
[ Master bug#12684] The format buttons are missing from the stacked area chart on the dashboard if the language is not English

### PR DESCRIPTION
**Pull request by s7design**
Hi @dvuckovic,

Herby is solved translation of the controls labels in stats. There is used Core.Config.Get instead of Core.Language.Translate because  Dev::Tools::TranslationsUpdate doesn't check thirdparty  JavaScript files.
There is also translate Format name in preview screen. 

Regards
Zoran

### CODE REVIEW

File: var/httpd/htdocs/js/thirdparty/nvd3-1.7.1/models/OTRSLineChart.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ nv.models.OTRSlineChart = function() {
 // OTRS
 // ---
 //       , noData = 'No Data Available.'
-        , noData = Core.Language.Translate('No Data Available.')
+        , noData = Core.Config.Get('NoDataAvailable')
```
Comment (dvuckovic) [inline var/httpd/htdocs/js/thirdparty/nvd3-1.7.1/models/OTRSLineChart.js]: @zottto We would need to go back to old API for translations in this case. The reason is that `TranslationsUpdate` console command will not mark strings translatable for JS in thirdparty files, only core. What do you think, is this acceptable? Translations would be set by Perl `AddJSData()`, of course.

Comment (mrcbnsls) [inline var/httpd/htdocs/js/thirdparty/nvd3-1.7.1/models/OTRSLineChart.js]: @dvuckovic, as of today, the TranslationsUpdate command will also scan 3rd party files (which contain custom markers), so we should go back to use Core.Language.Translate there.

**Comment by s7design** [comment]
Hi @dvuckovic 

This PR is rebased because of changes that has been done on TranslatUpdate. There was left only translate in Preview screen for Format name of graph stats. 

There is another PR for OTRS5.

Regards
Zoran

**Comment by dvuckovic** [comment]
Merged in 4e20d01, thanks @s7design!

**Pull request closed without merging**